src/Uno.UI.XamlHost.Skia.Wpf/UnoXamlHostBase.Focus.cs

```diff
@@ -142,7 +142,7 @@ private static WF.Rect BoundsRelativeTo(FrameworkElement sibling1, System.Window
             return origin;
         }
 
-        private bool _onTakeFocusRequested = false;
+        //private bool _onTakeFocusRequested;
```

Comment:
Commented this out instead of completely deleting because the code that reads the field is commented as well.

https://github.com/unoplatform/uno/blob/1198d47de79caff733d94d5c01b7df677d85168c/src/Uno.UI.XamlHost.Skia.Wpf/UnoXamlHostBase.Focus.cs#L83

Reply:
Looks good.